sources/packages/api/src-sql/DisbursementSchedules/Add-coe-cols.sql

```diff
@@ -0,0 +1,25 @@
+-- Add coe_approved to disbursement_schedules table.
+ALTER TABLE
+    sims.disbursement_schedules
+ADD
+    COLUMN IF NOT EXISTS coe_approved BOOLEAN;
```

Comment:
Is the Boolean value enough to control the COE?
I was under the impression that the application COE status would be "moved" to the disbursement column.

Reply:
During backlog grooming we decided to have boolean, where NULL represents Required, true as Completed and false as Denied.

Reply:
@abschwenker please confirm in the future, do we have the use of Not Required in COEStatus? We are aware that currently we are not using it.

Reply:
After discussion with @abschwenker and @andrewsignori-aot column of type enum is now used instead of boolean considering the extensibility offered by enum.

Reply:
@guru-aot, as per the conversation with @abschwenker, "Not required" is not needed in the COE Status.